app/presenters/step_by_step_page_presenter.rb

```diff
@@ -108,4 +117,27 @@ def last_saved
   def assigned?
     step_by_step_page.assigned_to.present?
   end
+
+private
+
+  def steps
+    step_number = 0
+    step_by_step_page.steps.map do |step|
+      step_number = step_number + 1 if step.logic == 'number'
+      {
+        field: step.logic == 'number' ? step_number : step.logic,
+        value: "#{step.title} #{broken_link_status_text(step)}", # TODO: render broken-links-summary.html
```

Comment:
Want to do something like this:

```ruby
value: "#{step.title} #{render 'shared/steps/broken-links-summary', step: step}"
```

...but `render` is not available to the presenter.